### PR TITLE
sell tcl strat rewards, adjust slippage/deadline

### PR DESCRIPTION
**Pull request by jalbrekt85**
handles #1117

### CODE REVIEW

File: scripts/issue/1117/sell_tcl_rewards.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    # https://curve.readthedocs.io/exchange-lp-tokens.html#CurveToken.approve
+    crv.approve(vault.cow.vault_relayer, 0)
+
+    vault.cow.market_sell(
+        crv,
+        dai,
+        crv.balanceOf(vault),
+        deadline=DEADLINE,
+        coef=COEF,
```
Comment (gosuto-inzasheru) [inline scripts/issue/1117/sell_tcl_rewards.py:39]: if you remove the last comma in a list such as this one, the linter will try to fit it into one line. if you leave it, it forces one item per line.

hence the diff between this line and L27/29

just saying :)

**Review by gosuto-inzasheru — APPROVED** [review]
perfecto

**Review by petrovska-petro — APPROVED** [review]
deadline bump up and free up bit more the slippage ✅

File: scripts/issue/1117/sell_tcl_rewards.py (hunk 1 above)
Comment (jalbrekt85) [reply scripts/issue/1117/sell_tcl_rewards.py:39]: ah

**Pull request merged**